package.json

```diff
@@ -17,7 +17,7 @@
     "lint": "prettier --check 'src/**/*.ts'",
     "publish": "pika publish",
     "version": "npm run build",
-    "test": "jest __tests__/runner.js",
+    "test": "jest __tests__/runner.js --test-timeout=10000",
```

Comment:
Also just FYI: bumped the Jest test timeout from 5s (default) to 10s. With nomodule, we’re doing 2 passes on Rollup and it was basically impossible for CI to finish both within 5s.

Reply:
Edit: 15s (it’s slower on GitHub Actions 😅 )